docs/blog/2019-08-13-Localised-strings-from-Kentico-Cloud-GatsbyJS-GraphQL/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Adding and retrieving localised strings from Kentico Cloud to GatsbyJS and GraphQL
```

Comment:
To match the [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#prefer-us-english) it seems like we should probably opt for the US style "localized" instead:

```suggestion
title: Adding and retrieving localized strings from Kentico Cloud to GatsbyJS and GraphQL
```

Reply:
Thanks I will change this